dataactcore/config_example.yml

```diff
@@ -68,6 +68,10 @@ broker:
     # AWS credentials for uploading files. Ignored if use_aws is false.
     aws_create_temp_credentials: true
 
+    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
+    sf_133_folder: config
+    sf_133_file_one: sf_133_part1.csv
+    sf_133_file_two: sf_133_part2.csv
```

Comment:
I thought about combining them, but don't many people (myself included) generally open csv files in Excel?  I'll try combining and see if that still breaks Excel in csv.